--- Portfile.orig	2015-03-26 23:32:04.000000000 -0500
+++ Portfile	2015-04-14 07:42:55.000000000 -0500
@@ -163,18 +163,43 @@
     depends_lib-append port:lcms
 }
 
-# Option to check MySQL availability before compiling target
+# Option to enable internal MySQL server
 
-variant mysql_check description {Check MySQL availability before building digiKam} {}
+variant mysql_internal description {Enable support for internal MySQL database} {}
 
-if {[variant_isset mysql_check]} {
+if {[variant_isset mysql_internal]} {
     configure.args-append -DENABLE_INTERNALMYSQL=on
-
-    depends_lib-append    path:bin/mysql_config5:mysql5 \
+    depends_lib-append    path:bin/mysql_config:mysql5 \
 } else {
     configure.args-append -DENABLE_INTERNALMYSQL=off
 }
 
+# Options to build external MySQL database support
+
+variant mysql51_external \
+    description {Enable external MySQL 5.1 database support} {
+    configure.args-append -DENABLE_MYSQLSUPPORT=ON
+    depends_lib-append    port:qt4-mac-mysql51-plugin
+}
+
+variant mysql55_external \
+    description {Enable external MySQL 5.5 database support} {
+    configure.args-append -DENABLE_MYSQLSUPPORT=ON
+    depends_lib-append    port:qt4-mac-mysql55-plugin
+}
+
+variant mysql56_external \
+    description {Enable external MySQL 5.6 database support} {
+    configure.args-append -DENABLE_MYSQLSUPPORT=ON
+    depends_lib-append    port:qt4-mac-mysql56-plugin
+}
+
+variant mariadb_external \
+    description {Enable external MySQL database support using MariaDB 5.5} {
+    configure.args-append -DENABLE_MYSQLSUPPORT=ON
+    depends_lib-append    port:qt4-mac-mariadb-plugin
+}
+
 livecheck.type      regex
 livecheck.url       http://kde-apps.org/content/show.php/digiKam+Software+Collection?content=137059
 livecheck.regex     "digiKam Software Collection.*(4\\.\\d+?(\\.\\d+))"
