diff --git a/CMakeLists.txt b/CMakeLists.txt
index e5cee1b4a..bd36b899e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -192,7 +192,7 @@ elseif(APPLE)
       # for Mac OS X, everything is put inside an application bundle
       SET (CMAKE_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX}/)
       # path for library references
-      SET (CMAKE_INSTALL_NAME_DIR @executable_path/lib)
+      SET (CMAKE_INSTALL_NAME_DIR ${CMAKE_INSTALL_PREFIX}/lib)
       # install the Info.plist file
       install(FILES src/mac/Info.plist DESTINATION ${CMAKE_INSTALL_PREFIX}/Marble.app/Contents)
       #SET (lib_dir ${CMAKE_INSTALL_PREFIX}/Marble.app/Contents/MacOS/lib)
diff --git a/src/lib/marble/ViewportParams.cpp b/src/lib/marble/ViewportParams.cpp
index 58715f5a8..d9e31125b 100644
--- a/src/lib/marble/ViewportParams.cpp
+++ b/src/lib/marble/ViewportParams.cpp
@@ -96,7 +96,7 @@ ViewportParamsPrivate::ViewportParamsPrivate( Projection projection,
       m_planetAxis(),
       m_planetAxisMatrix(),
       m_radius( radius ),
-      m_angularResolution(4.0 / abs(m_radius)),
+      m_angularResolution(4.0 / qAbs(m_radius)),
       m_size( size ),
       m_dirtyBox( true ),
       m_viewLatLonAltBox()
