diff --git a/autotests/jobtest.h b/autotests/jobtest.h
index 5687b4a..5001309 100644
--- a/autotests/jobtest.h
+++ b/autotests/jobtest.h
@@ -24,6 +24,8 @@
 #include <QtCore/QObject>
 #include <kio/job.h>
 
+#undef stat
+
 class JobTest : public QObject
 {
     Q_OBJECT
diff --git a/src/core/slaveinterface.h b/src/core/slaveinterface.h
index f803a19..6e9acc7 100644
--- a/src/core/slaveinterface.h
+++ b/src/core/slaveinterface.h
@@ -31,6 +31,8 @@
 
 class QUrl;
 
+#undef open
+
 namespace KIO
 {
 
