diff --git a/src/QtAV/AVClock.h b/src/QtAV/AVClock.h
index 980e3b1..b3d386d 100644
--- a/src/QtAV/AVClock.h
+++ b/src/QtAV/AVClock.h
@@ -110,13 +110,13 @@ public:
      */
     bool syncEndOnce(int id);
 
-signals:
+Q_SIGNALS:
     void paused(bool);
     void paused(); //equals to paused(true)
     void resumed();//equals to paused(false)
     void started();
     void resetted();
-public slots:
+public Q_SLOTS:
     //these slots are not frequently used. so not inline
     /*start the external clock*/
     void start();
diff --git a/src/QtAV/AVDemuxer.h b/src/QtAV/AVDemuxer.h
index cdc3441..3b720f5 100644
--- a/src/QtAV/AVDemuxer.h
+++ b/src/QtAV/AVDemuxer.h
@@ -187,7 +187,7 @@ public:
      */
     void setOptions(const QVariantHash &dict);
     QVariantHash options() const;
-signals:
+Q_SIGNALS:
     void unloaded();
     void userInterrupted(); //NO direct connection because it's emit before interrupted happens
     void loaded();
diff --git a/src/QtAV/AVPlayer.h b/src/QtAV/AVPlayer.h
index c0525f8..5317012 100644
--- a/src/QtAV/AVPlayer.h
+++ b/src/QtAV/AVPlayer.h
@@ -388,7 +388,7 @@ public:
     MediaEndAction mediaEndAction() const;
     void setMediaEndAction(MediaEndAction value);
 
-public slots:
+public Q_SLOTS:
     /*!
      * \brief load
      * Load the current media set by setFile(); Can be used to reload a media and call play() later. If already loaded, does nothing and return true.
diff --git a/src/QtAV/Filter.h b/src/QtAV/Filter.h
index cc5a94f..9914533 100644
--- a/src/QtAV/Filter.h
+++ b/src/QtAV/Filter.h
@@ -62,7 +62,7 @@ public:
     bool uninstall();
 public Q_SLOTS:
     void setEnabled(bool enabled = true);
-signals:
+Q_SIGNALS:
     void enabledChanged(bool);
 protected:
     /*
diff --git a/src/QtAV/OpenGLVideo.h b/src/QtAV/OpenGLVideo.h
index 986b39f..b4e7f60 100644
--- a/src/QtAV/OpenGLVideo.h
+++ b/src/QtAV/OpenGLVideo.h
@@ -102,7 +102,7 @@ Q_SIGNALS:
 protected:
     DPTR_DECLARE(OpenGLVideo)
 
-private slots:
+private Q_SLOTS:
     /* used by Qt5 whose QOpenGLContext is QObject and we can call this when context is about to destroy.
      * shader manager and material will be reset
      */
diff --git a/src/QtAV/Subtitle.h b/src/QtAV/Subtitle.h
index 2d1c230..68f044a 100644
--- a/src/QtAV/Subtitle.h
+++ b/src/QtAV/Subtitle.h
@@ -179,7 +179,7 @@ public:
     void setFontsDir(const QString& value);
     bool isFontFileForced() const;
     void setFontFileForced(bool value);
-public slots:
+public Q_SLOTS:
     /*!
      * \brief start
      * start to process the whole subtitle content in a thread
@@ -187,7 +187,7 @@ public slots:
     void load();
     void loadAsync();
     void setTimestamp(qreal t);
-signals:
+Q_SIGNALS:
     // TODO: also add to AVPlayer?
     /// empty path if load from raw data
     void loaded(const QString& path = QString());
diff --git a/src/QtAV/SubtitleFilter.h b/src/QtAV/SubtitleFilter.h
index bd3faa0..332bc7b 100644
--- a/src/QtAV/SubtitleFilter.h
+++ b/src/QtAV/SubtitleFilter.h
@@ -83,15 +83,15 @@ public:
     QFont font() const;
     void setColor(const QColor& c);
     QColor color() const;
-public slots:
+public Q_SLOTS:
     // TODO: enable changed & autoload=> load
     void setAutoLoad(bool value);
-signals:
+Q_SIGNALS:
     void rectChanged();
     void fontChanged();
     void colorChanged();
     void autoLoadChanged(bool value);
-signals:
+Q_SIGNALS:
     void fileChanged();
     void canRenderChanged();
     void loaded(const QString& path);
diff --git a/src/QtAV/VideoCapture.h b/src/QtAV/VideoCapture.h
index 61a3cef..bba49a4 100644
--- a/src/QtAV/VideoCapture.h
+++ b/src/QtAV/VideoCapture.h
@@ -116,7 +116,7 @@ Q_SIGNALS:
     void qualityChanged();
     void captureNameChanged();
     void captureDirChanged();
-private slots:
+private Q_SLOTS:
     void handleAppQuit();
 private:
     void setVideoFrame(const VideoFrame& frame);
