Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 4688)
+++ CMakeLists.txt	(working copy)
@@ -30,9 +30,9 @@
 SET( PACKAGE_BUGREPORT      "ahuggel@gmx.net" )
 SET( PACKAGE                "exiv2" )
 SET( PACKAGE_NAME           "exiv2" )
-SET( PACKAGE_STRING         "exiv2 0.25" )
+SET( PACKAGE_STRING         "exiv2 0.26-svn" )
 SET( PACKAGE_TARNAME        "exiv2" )
-SET( PACKAGE_VERSION        "0.25" )
+SET( PACKAGE_VERSION        "0.26" )
 SET( PACKAGE_URL            "http://www.exiv2.org" )
 SET( GENERIC_LIB_VERSION    "14.0.0" )
 SET( GENERIC_LIB_SOVERSION  "14" )
