Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 4688)
+++ CMakeLists.txt	(working copy)
@@ -116,7 +116,7 @@
 IF( EXIV2_ENABLE_XMP )
     SET( HAVE_XMP_TOOLKIT ON )
     SET( XMPLIB "do" )
-    set(THREADS_PREFER_PTHREAD_FLAG ON)
+#    set(THREADS_PREFER_PTHREAD_FLAG ON)
     find_package(Threads REQUIRED)
 ENDIF( EXIV2_ENABLE_XMP )
 
