Index: bootstrap.macports
===================================================================
--- bootstrap.macports	(revision 4686)
+++ bootstrap.macports	(working copy)
@@ -63,17 +63,17 @@
       -DEXIV2_ENABLE_SHARED=ON \
       -DEXIV2_ENABLE_XMP=ON \
       -DEXIV2_ENABLE_LIBXMP=ON \
-      -DEXIV2_ENABLE_VIDEO=ON \
       -DEXIV2_ENABLE_PNG=ON \
-      -DEXIV2_ENABLE_NLS=ON \
+      -DEXIV2_ENABLE_NLS=OFF \
       -DEXIV2_ENABLE_PRINTUCS2=ON \
       -DEXIV2_ENABLE_LENSDATA=ON \
       -DEXIV2_ENABLE_COMMERCIAL=OFF \
-      -DEXIV2_ENABLE_BUILD_SAMPLES=ON \
-      -DEXIV2_ENABLE_BUILD_PO=ON \
-      -DEXIV2_ENABLE_VIDEO=ON \
-      -DEXIV2_ENABLE_WEBREADY=ON \
-      -DEXIV2_ENABLE_CURL=ON \
-      -DEXIV2_ENABLE_SSH=ON \
+      -DEXIV2_ENABLE_BUILD_SAMPLES=OFF \
+      -DEXIV2_ENABLE_BUILD_PO=OFF \
+      -DEXIV2_ENABLE_VIDEO=OFF \
+      -DEXIV2_ENABLE_WEBREADY=OFF \
+      -DEXIV2_ENABLE_CURL=OFF \
+      -DEXIV2_ENABLE_SSH=OFF \
+      -DEXV_WANT_VISIBILITY_SUPPORT=ON \
       -Wno-dev \
       ..
